test/js/transform-value.js

```diff
@@ -9,14 +9,21 @@ suite('TransformValue', function() {
 
   test('TransformValue constructor throws exception for invalid types',
       function() {
-    assert.throws(function() {new TransformValue()});
+    assert.throws(function() {new TransformValue(null)});
     assert.throws(function() {new TransformValue({})});
-    assert.throws(function() {new TransformValue([])});
     assert.throws(function() {new TransformValue(['1', '2'])});
     assert.throws(function() {new TransformValue([null])});
     assert.throws(function() {new TransformValue([new NumberValue(5)])});
   });
 
+  test('TransformValue empty constructor should create an object with ' + 
+    'the expected properties', function() {
```

Comment:
Maybe say that it should represent a 2D identity matrix with an empty cssString?